src/main/java/ui/map/MapView.java

```diff
@@ -7,32 +7,51 @@
 import javafx.geometry.Pos;
 import javafx.scene.control.Button;
 import javafx.scene.control.ComboBox;
+import javafx.scene.control.Label;
 import javafx.scene.layout.BorderPane;
 import javafx.scene.layout.VBox;
+import manager.LanguageUtility;
+import model.interfaces.WithToStringProperty;
 import model.map.Map;
+import model.map.specification.CryptSpecification;
 import ui.IView;
 import ui.ViewPart;
+import ui.part.UpdatingListCell;
+
+import java.util.Random;
+
+import static manager.LanguageUtility.getMessageProperty;
+import static ui.ViewFactory.labelTextField;
 
 public class MapView extends ViewPart {
 
     private final MapCanvas canvas;
-    private ObjectProperty<Map> map;
-    private ObjectProperty<Structure> selectedStructure;
+    private final ObjectProperty<Map> map;
+    private final ObjectProperty<Structure> selectedStructure;
+    private final LongProperty seed;
+    private final IntegerProperty width, height, depth;
     private final BooleanProperty loading;
     private final IntegerProperty shownYLayer;
+    private final Random random;
 
     public MapView(IView parent) {
-        super("Karten", parent);
+        super("map.title", parent);
         this.selectedStructure = new SimpleObjectProperty<>();
         this.loading = new SimpleBooleanProperty(false);
         this.shownYLayer = new SimpleIntegerProperty(0);
         this.map = new SimpleObjectProperty<>(null);
+        this.seed = new SimpleLongProperty(5411351666781167994L);
```

Comment:
You should use a random seed by default here

Reply:
This is work in progress.
This is a nice test seed.